### PR TITLE
Font size increased for login and sign up heading #43

### PR DESCRIPTION
**Pull request by Mayank-Jain-1**
## Description
The "Login" and "Sign Up" heading were changed from `<h3>` tag to `<h1>` tag due to there small size.

## Issue ticket number and link
Issue number  [issue #43](https://github.com/iScsc/iscsc.fr/issues/43) by @mayank-jain-1

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I had run tests and found no visual error.

### CODE REVIEW

**Review by atxr — APPROVED** [review]
Looks good :wink: 
I put some screenshot bellow to see the change

Before:
image


After:
image

I'll wait for @ctmbl approval :rocket: 
Thanks for your contribution!

**Review by ctmbl — APPROVED** [review]
It actually looks better you're right!
Thanks for your contribution! It really means a lot to us :heart:

**Comment by Mayank-Jain-1** [comment]
> It actually looks better you're right! Thanks for your contribution! It really means a lot to us ❤️
I am happy to help. Would be looking further for any issues i can help with.

**Pull request merged**
